### ISSUE TITLE
Disable text input

### ISSUE THREAD

**Issue by vielleicht** [daattali/shinyjs#44]
In order to improve display on mobile devices, disabling text input would be a nice enhancement. The palette fold-out should therefore be triggered by a button.

**Comment by daattali** [comment]
I'm sorry but I don't understand the problem/request. There is already a `disable` function that can be used on text inputs.  Can you rephrase and go into more detail?

**Comment by daattali** [comment]
Closing due to inactivity and not describing a problem. Feel free to reopen if you need

**Comment by vielleicht** [comment]
Ahh, sorry.
Indeed, there is a disable function, which just disables the text output in the text field. The field is then empty.

My suggestion was to exchange the text field with and a button, when the text is disabled. Clicking the button (instead of the text field) whould then lead to the color palette fold-out.

**Comment by daattali** [comment]
I'm still very unsure what you mean...  Are you talking about `colourInput`? Because your original post said nothing about colours.
